__tests__/app.test.js

```diff
@@ -235,3 +235,56 @@ describe('api/articles', () => {
     });
   });
 });
+
+describe('api/articles/:article_id/comments', () => {
+  describe('GET - Successful Responses', () => {
+    test('status: 200 and responds with an array of the comments for a given article with the required properties', () => {
+      return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          console.log(comments);
+          expect(comments).toHaveLength(11);
+          comments.forEach((article) =>
+            expect(article).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+              })
+            )
+          );
+        });
+    });
+  });
+  describe('GET - Error Responses', () => {
+    test('status: 400 when given an invalid id', () => {
+      return request(app)
+        .get('/api/articles/banana/comments')
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request!');
+        });
+    });
+    test('status: 404 and comments not found message', () => {
```

Comment:
Is this an article that is valid with no comments? Would we expect a 404 for that? A new article would be added to the database with no comments, so that would error. Consider what this status code should be.

Reply:
I think I understand, if it was a valid article with no comments then it would just be a 200 response with an empty array. I have edited the code to reflect this and the only 404 error is now if its a valid article input which does not exist.